Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,588 @@
+    :return: str
+    """
+    p = urllib.parse.urlparse(url, protocol)  # type: ignore
+    netloc = p.netloc or p.path
+    path = p.path if p.netloc else ''
+    if not netloc.startswith('www.') and www:
+        netloc = 'www.' + netloc
+    p = urllib.parse.ParseResult(protocol, netloc, path, *p[3:])  # type: ignore
+    return p.geturl()
+
+
+def is_valid_url(url: str) -> Tuple[bool, str]:
+    """
+    Check is an url is valid by requesting it using different protocol
+    :param url: url
+    :return: bool
+    """
+    try:
+        response = requests.get(url, verify=False)  # nosec  guardrails-disable-line
+    except requests.exceptions.RequestException:
+        prepend_url = prepend_protocol(url, 'http', True)
+        try:
+            response = requests.get(prepend_url, verify=False)  # nosec guardrails-disable-line
+        except requests.exceptions.RequestException:
+            prepend_url = prepend_protocol(url, 'https', True)
+            try:
+                response = requests.get(prepend_url, verify=False)  # nosec guardrails-disable-line
+            except requests.exceptions.RequestException:
+                prepend_url = prepend_protocol(url, 'http', False)
+                try:
+                    response = requests.get(prepend_url, verify=False)  # nosec guardrails-disable-line
+                except requests.exceptions.RequestException:
+                    prepend_url = prepend_protocol(url, 'https', False)
+                    try:
+                        response = requests.get(prepend_url, verify=False)  # nosec guardrails-disable-line
+                    except requests.exceptions.RequestException:
+                        return False, MSG_IMPOSSIBLE_CONNECTION
+    if response.status_code == 200:
+        return True, EMPTY_STRING
+    else:
+        return False, response.reason
+
+
+def return_entry_summary(pred_json: Dict, url: str, whitelist: bool, output_rasterize: Dict):
+    """
+    Return entry to demisto
+    :param pred_json: json with output of the model
+    :param url: url
+    :param whitelist: if url belongs to whitelist of the model
+    :return: entry to demisto
+    """
+    if not pred_json:
+        return
+    if whitelist:
+        url_score = SCORE_BENIGN
+        url_score = GREEN_COLOR % str(url_score) if url_score < SCORE_THRESHOLD else RED_COLOR % str(
+            url_score)
+    else:
+        url_score = round(pred_json[MODEL_KEY_URL_SCORE], 2)
+        url_score = GREEN_COLOR % str(url_score) if url_score < SCORE_THRESHOLD else RED_COLOR % str(url_score)  # type: ignore
+    pred_json_colored = get_colored_pred_json(pred_json)
+    domain = extract_domainv2(url)
+    explain = {
+        KEY_CONTENT_DOMAIN: domain,
+        KEY_CONTENT_URL: url,
+        KEY_CONTENT_LOGO: str(pred_json[MODEL_KEY_LOGO_FOUND]),
+        KEY_CONTENT_LOGIN: str(pred_json[MODEL_KEY_LOGIN_FORM]),
+        KEY_CONTENT_URL_SCORE: url_score,
```

Comment:
also here, add `URLScore` and `Final Verdict` outputs which are not HR

Reply:
no it's 2 different entries. we can talk about it